### PR TITLE
Add initial default CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by brianwarner**
This is the initial commit of a default CONTRIBUTING.md file, which will
replicate across all repositories which do not currently have a local
CONTRIBUTING.md.

This PR should be approved by the TSC prior to being merged.

Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+| Andrii Rosa       |[arhimondr](https://github.com/arhimondr)          | Facebook    |
+| Rebecca Schlussel |[rschlussel](https://.github.com/rschlussel)       | Facebook    |
+| James Sun         |[highker](https://github.com/highker)              | Facebook    |
+| Wenlei Xie        |[wenleix](https://github.com/wenleix)              | Facebook    |
+| Rongrong Zhong    |[rongrong](https://github.com/rongrong)            | Facebook    |
+| Shixuan Fan       |[shixuan-fan](https://github.com/shixuan-fan)      | Facebook    |
+| Jiexi Lin         |[jessesleeping](https://github.com/jessesleeping)  | Facebook    |
+
+
+The path to committership starts with becoming a contributor to the project. Over time contributors prove that they have high commitment to the project and they get nominated by an existing committer to become a committer.
```
Comment (caniszczyk) [inline CONTRIBUTING.md]: maybe add that a simple majority vote is needed to add an additional committer

Comment (brianwarner) [reply CONTRIBUTING.md]: Thanks for pointing this out, @caniszczyk. I attempted to clarify it, could you please have a look at what I just pushed?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,17 +12,34 @@ All Presto development occurs on [GitHub](https://github.com/prestodb), followin
...
+### Committers
 
 Committers are members of the contributor community who have demonstrated an ongoing commitment to the health and sustainability of the Presto project through good citizenship and wise stewardship. These individuals have been elevated to a decision-making role, and responsible for reviewing and merging code from contributors. They may also be involved in the release process or other critical project roles.
 
-* [Current list of Presto committers](https://github.com/prestodb/presto/wiki/committers)
+The path to committership starts with becoming a contributor to the project. Over time contributors prove that they have high commitment to the project and they get nominated by an existing committer to become a committer.
+
+When a committer candidate is nominated by an existing committer lazy consensus is required (no vetoes) from existing committers. The nomination should include a justification of why that person should become a committer (e.g., code/test/doc contributions, other positive impact to the project & community, etc.).
+
+Committer candidates:
```
Comment (nezihyigitbasi) [inline CONTRIBUTING.md:42]: Now that we specify committer criteria in CONTRIBUTING.md file, do we still want to keep the same/similar info in the [wiki](https://github.com/prestodb/presto/wiki/How-to-become-a-Presto-committer%3F)?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,17 +12,34 @@ All Presto development occurs on [GitHub](https://github.com/prestodb), followin
...
+
+* Demonstrate good collaboration skills
+* Submit high quality pull requests that conform to Presto coding styles and best practices.
+* Demonstrate good understanding of Presto’s criteria for accepting pull requests
+* Help with reviewing pull requests from the community
+* Contribute to design/technical discussions on Github and Slack
+* Help with user issues on Github, user mailing list, and Slack
+have good judgement around when some code should be merged, or when to ask someone else to make that judgement
```
Comment (nezihyigitbasi) [inline CONTRIBUTING.md]: Missing `*`.

Comment (brianwarner) [reply CONTRIBUTING.md]: Thanks @nezihyigitbasi.  I'll push a patch with that correction right now.

File: CONTRIBUTING.md (hunk 2 above)
Comment (brianwarner) [reply CONTRIBUTING.md:42]: It's a good question.  I have a few more things to add to the agenda for tomorrow's meeting, I can add this as a topic.

Comment (caniszczyk) [reply CONTRIBUTING.md:42]: I'd drop the wiki for listing this type of stuff

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+| Masha Basmanova   |[mbasmanova](https://github.com/mbasmanova)        | Facebook    |
+| Zhenxiao Luo      |[zhenxiao](https://github.com/zhenxiao)            | Twitter     |
+| Andrii Rosa       |[arhimondr](https://github.com/arhimondr)          | Facebook    |
+| Rebecca Schlussel |[rschlussel](https://.github.com/rschlussel)       | Facebook    |
+| James Sun         |[highker](https://github.com/highker)              | Facebook    |
+| Wenlei Xie        |[wenleix](https://github.com/wenleix)              | Facebook    |
+| Rongrong Zhong    |[rongrong](https://github.com/rongrong)            | Facebook    |
+| Shixuan Fan       |[shixuan-fan](https://github.com/shixuan-fan)      | Facebook    |
+| Jiexi Lin         |[jessesleeping](https://github.com/jessesleeping)  | Facebook    |
+
```
Comment (rongrong) [inline CONTRIBUTING.md:37]: We recently just added a new committer:
``` 
| Leiqing Cai  | [caithagoras](https://github.com/caithagoras) | Facebook |
```
Could you please add him in. Thanks!

Comment (brianwarner) [reply CONTRIBUTING.md:37]: Thanks @rongrong, will do!  I'll push a patch now for that, and also adding Venki to the TSC.

**Comment by galemccommons** [comment]
Merging. Thanks @brianwarner

**Pull request merged**